afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+late	-1
+later	-1
+ganske	1
+forhindre	-1
+forhindret	-1
+stikk	-2
+fengsel	-2
+fange	-2
+fanger	-2
+privilegert	2
+proaktiv	2
+problemer	-2
+profitt	2
+lønnsom	2
+profitør	-2
+fortjeneste	2
+framgang	2
+forbyr	-1
+fremtredende	1
+love	1
+lovet	1
+løfter	1
+reklamere	1
+forfremmet	1
+fremmer	1
+fremme	1
+omgå	1
+tiltale	-1
+prospektet	1
+prospekter	1
+velstand	3
+velstående	3
+beskytte	1
+beskyttet	1
+beskytter	1
+demonstranter	-2
+protesterer	-2
+protester	-2
+stolt	2
+provosere	-1
+provosert	-1
+provoserer	-1
+vekk	-1
+klokskap	2
+pseudovitenskap	-3
+psykopatisk	-2
+reneste	1
+målbevisst	2
+skjelv	-2
+kvaliteter	2
+kvalitet	2
+avhør	-1
+rasisme	-3
+rasist	-3
+rasister	-3
+regn	-1
+tirade	-3
+ranters	-3
+voldtekt	-4
+voldtatt	-4
+voldtektsmann	-4
+henrykkelse	2
+opprykket	2
+begeistrede	4
+utslett	-2
+ratifisert	2
+å nå	1
+nådd	1
+når	1
+nå	1
+forsikre	1
+forsikrer	1
+betryggende	2
+opprører	-2
+opprørerne	-2
+resesjon	-2
+uforsvarlig	-2
+erkjennelse	2
+anbefales	2
+anbefaler	2
+innløst	2
+raffinere	1
+raffinert	1
+foredler	1
+forfriskende	2
+nekte	-2
+nektet	-2
+nekter	-2
+angre	-2
+angrende	-2
+angrer	-2
+angret	-2
+reigning	1
+avvise	-1
+avvisning	-2
+avviser	-1
+fryde	4
+frydet	4
+jubler	4
+gleder seg	4
+avslappet	2
+nådeløse	-1
+pålitelighet	2
+pålitelig	2
+avhengige	2
+lindre	1
+lettet	2
+lindrer	1
+lindrende	2
+samvittighetsnag	-2
+frastøtende	-2
+ettervirkning	-2
+konsekvenser	-2
+reprimande	-2
+irettesettelser	-2
+repulse	-1
+frastøtt	-2
+redde	2
+reddet	2
+redder	2
+ergerlig	-2
+si opp	-1
+gikk av	-1
+trekker seg	-1
+resolutt	2
+vedtak	2
+løse	2
+løst	1
+løser	1
+løse	1
+respekt	2
+respektert	2
+henseender	2
+ansvar	1
+ansvarlig	2
+mottakelig	2
+avslappende	2
+rastløs	-2
+restaurere	1
+restaurert	1
+gjenoppretting	1
+gjenopprette	1
+begrense	-2
+restriktiv	-1
+begrenser	-2
+beholdes	-1
+tilbakestående	-2
+retrett	-1
+hevngjerrig	-2
+ærede	2
+gjenopplive	2
+gjenoppliver	2
+belønning	2
+belønnet	2
+givende	2
+belønninger	2
+rik	2
+rikt	2
+rigg	-1
+rigget	-1
+riktig retning	3
+rettmessige	2
+rettmessig	2
+streng	3
+opptøyer	-2
+stige	1
+reiser seg	1
+fare	-2
+risikoer	-2
+risikabelt	-2
+medrivende	3
+rane	-2
+raner	-2
+robs	-2
+roflmao	4
+romanse	2
+kjærester	2
+rotfl	4
+råtten	-3
+uhøflig	-2
+ruinert	-2
+ruiner	-2
+sabotasje	-2
+lei seg	-2
+trist	-2
+dessverre	-2
+trygt	1
+sikrere	2
+sikkerhet	1
+honnør	2
+hilste	2
+salutter	2
+frelse	2
+sarkastisk	-2
+lagre	2
+lagret	2
+sparing	1
+skandale	-3
+skandaløs	-3
+skandalene	-3
+syndebukk	-2
+syndebukker	-2
+arr	-2
+skummelt	-2
+skeptisk	-2
+skjenne	-2
+kule	3
+hånlig	-2
+hyle	-2
+skrek	-2
+skrudd	-2
+feilet	-3
+avskum	-3
+sømløs	2
+sømløst	2
+sikre	2
+sikret	2
+oppvigleri	-2
+opprørsk	-2
+forført	-1
+selvbesmittelse	-2
+selvsikkerhet	2
+selvmotsigende	-2
+selv villedet	-2
+selvopptatt	-3
+egoisme	-3
+setning	-2
+setninger	-2
+utmålingen	-2
+bosetting	1
+bosetninger	1
+alvorlig	-2
+kjønnsdiskriminerende	-2
+skjelvende	-2
+skam	-2
+skjemmes	-2
+skammelig	-2
+dele	1
+delt	1
+aksjer	1
```

Comment:
I suppose that in the normal sense this word has usually little sentiment value to it.

Reply:
I'll remove this one